src/Bridge/Laravel/Console/Commands/LogUnindexedSearchItemCommand.php

```diff
@@ -0,0 +1,56 @@
+<?php
+declare(strict_types=1);
+
+namespace LoyaltyCorp\Auditing\Bridge\Laravel\Console\Commands;
+
+use Illuminate\Console\Command;
+use LoyaltyCorp\Auditing\DataTransferObjects\LogLine;
+use LoyaltyCorp\Auditing\Interfaces\Services\LogWriterInterface;
+use LoyaltyCorp\Auditing\Interfaces\Services\SearchLogWriterInterface;
+
+final class LogUnindexedSearchItemCommand extends Command
+{
+    /**
+     * Construct command to index search items.
+     */
+    public function __construct()
+    {
+        $this->description = 'Log items not indexed in search.';
+        $this->signature = 'auditing:log:search-items';
+
+        parent::__construct();
+    }
+
+    /**
+     * Handle command.
+     *
+     * @param \LoyaltyCorp\Auditing\Interfaces\Services\LogWriterInterface $logWriter
+     * @param \LoyaltyCorp\Auditing\Interfaces\Services\SearchLogWriterInterface $searchLogWriter
+     *
+     * @return void
+     *
+     * @throws \Exception Exception from DateTime if any
+     */
+    public function handle(
+        LogWriterInterface $logWriter,
+        SearchLogWriterInterface $searchLogWriter
+    ): void {
+        $this->info('Indexing log items for search...');
```

Comment:
I don't think you can do this.

There's no guarantee this entry and the one on line 54 will be next to each other. In fact, on a busy system, they won't be.

Reply:
Refactored. Created job to write to search.